account_invoice_section_sale_order/models/sale_order.py

```diff
@@ -16,34 +16,36 @@ def _create_invoices(self, grouped=False, final=False, date=None):
         the group name.
         Only do this for invoices targetting multiple groups
         """
+        # the context key 'current_line_sequence' is used by
+        # sale.order.line:_prepare_invoice_line to generate spaced apart sequence number
+        # for invoice lines. We use a mutable in the context because the context is an
+        # immutable dictionary: this way we can update the value of the first element on
+        # the list it and see the updated values in the different calls to
+        # _prepare_invoice_line
+        self = self.with_context(current_line_sequence=[10])
```

Comment:
```suggestion
        self = self.with_context(current_line_sequence=[10]).sorted(key=lambda order: order.date_order)
```
Oldest orders are shown in invoice first